src/avfoundation.cs

```diff
@@ -4703,10 +4867,15 @@ interface AVSampleCursor : NSCopying {
 		[Export ("currentSampleStorageRange")]
 		AVSampleCursorStorageRange CurrentSampleStorageRange { get; }
 
-#if MONOMAC
-		[NoTV, NoWatch, NoiOS, Mac (10,15)]
+#if MONOMAC || __MACCATALYST__
+		[NoTV, NoWatch, NoiOS, Mac (10,15), MacCatalyst (15,0)]
 		[Export ("currentSampleAudioDependencyInfo")]
 		AVSampleCursorAudioDependencyInfo CurrentSampleAudioDependencyInfo { get; }
+
+		[NullAllowed]
+		[NoWatch, NoTV, NoiOS, Mac (12, 0), MacCatalyst (15,0)]
```

Comment:
The type is already NoiOS, NoTV, NoWatch, so 3/5ths of this is not necessary.

The type is also MacCatalyst (15, 0), so there's another 1/5th which is unnecessary.

Same goes for the member just above.

That also means that the `#if MONOMAC || __MACCATALYST__` condition can be completely removed.